MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -3104,6 +3104,10 @@ public void setCurrentEdge(int e) {
         currentEdge = e;
     }
 
+    public void changeCurrentEdge(int amount) {
```

Comment:
Should `currentEdge` be allowed to dip below 0?

Reply:
Because of how this is used it cannot drop below 0. However... I could rework this into returning a boolean value and add it to the if statements instead of having the current check?

Reply:
I think the more straightforward change:
```
currentEdge = Math.max(currentEdge + amount, 0);
```
Also protects us from future areas that aren't so good at checking (I was thinking GM Edge, but I think that just 'sets' the edge).

Reply:
Implemented, and that does just set the edge.